src/code/z_parameter.c

```diff
@@ -1363,15 +1363,15 @@ u8 Item_Give(PlayState* play, u8 item) {
         osSyncPrintf(VT_RST);
 
         return ITEM_NONE;
-    } else if ((item >= ITEM_KOKIRI_EMERALD) && (item <= ITEM_ZORA_SAPPHIRE)) {
-        gSaveContext.inventory.questItems |= gBitFlags[item - ITEM_KOKIRI_EMERALD + QUEST_KOKIRI_EMERALD];
+    } else if ((item >= ITEM_SPSTONE_KOKIRI_EMERALD) && (item <= ITEM_SPSTONE_ZORA_SAPPHIRE)) {
+        gSaveContext.inventory.questItems |= gBitFlags[item - ITEM_SPSTONE_KOKIRI_EMERALD + QUEST_KOKIRI_EMERALD];
```

Comment:
Do you think I should? I feel like this would open a can of worms on defines

Reply:
I don't think "conversion" macros are inherently a bad idea, it would be a bit clearer. I assumed you would since you make defines for shifts that are sometimes only used once.

Reply:
Well let's say not this time then :p

I started the PR from looking at item icons in icon_item_static so it isn't the main focus here